pkg/sessionctx/variable/session.go

```diff
@@ -2801,6 +2801,9 @@ type Concurrency struct {
 
 	// IdleTransactionTimeout indicates the maximum time duration a transaction could be idle, unit is second.
 	IdleTransactionTimeout int
+
+	// BulkDMLParallelism indicates whether to enable pipelined transaction.
+	BulkDMLParallelism int
```

Comment:
There is no parallelism in current implementation. And renaming it won't break anything. So I suppose we don't to name it so aggressively. Can be a simple boolean